pandas/core/frame.py

```diff
@@ -8564,7 +8564,9 @@ def pivot(self, index=None, columns=None, values=None) -> DataFrame:
         margins : bool, default False
             Add all row / columns (e.g. for subtotal / grand totals).
         dropna : bool, default True
-            Do not include columns whose entries are all NaN.
+            Do not include columns whose entries are all NaN. If true:
```

Comment:
True refers to Python True value, we usually use it with capital T. Also, please avoid the colon, as it's inconsistent with the rest of the docs, and personally I think make things harder to read in this case.

Other than those small formatting things, I think your description is clear now, thanks for thr update.

Reply:
made the changes, thanks for the help.